### PR TITLE
added IB cases

### PR DESCRIPTION
**Pull request by zhaozhanqi**
auto sriov IB cases.

https://privatebin-it-iso.int.open.paas.redhat.com/?8e4e94a699b2c809#CfvybM9awcBqwo88ksGoVqtzoQpGLFNrXiwiyZ2NEzyY

@anuragthehatter @huiran0826  @weliang1 @rbbratta PTAL, thanks

### CODE REVIEW

**Comment by zhaozhanqi** [comment]
@huiran0826 @weliang1 @rbbratta PTAL, thanks

File: features/networking/ib-sriov.feature [hunk 1]
```diff
@@ -0,0 +1,158 @@
+Feature: Sriov IB related scenarios
+
+  # @author zzhao@redhat.com
```
Comment (weliang1) [inline features/networking/ib-sriov.feature:3]: Need a case_id here

File: features/networking/ib-sriov.feature [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+  @destructive
+  @admin
+  Scenario Outline: IPoIB and IB mode for CX4/CX5/CX6 card
+    Given the sriov operator is running well
+    Given I obtain test data file "networking/sriov/ib/<cardname>/<cardname>-ib.yaml"
```
Comment (weliang1) [inline features/networking/ib-sriov.feature:8]: Do we need a real car name or just <cardname> here?

File: features/networking/ib-sriov.feature (hunk 1 above)
Comment (zhaozhanqi) [reply features/networking/ib-sriov.feature:3]: this is `Outline` case, you can see the case id in line 63-65

File: features/networking/ib-sriov.feature (hunk 2 above)
Comment (zhaozhanqi) [reply features/networking/ib-sriov.feature:8]: yes, see line 63-65

File: features/networking/ib-sriov.feature (hunk 1 above)
Comment (weliang1) [reply features/networking/ib-sriov.feature:3]: Get it, thanks!

File: features/networking/ib-sriov.feature (hunk 2 above)
Comment (weliang1) [reply features/networking/ib-sriov.feature:8]: Get it, thanks!

**Comment by zhaozhanqi** [comment]
/assign @liangxia 
Could you help take look?

**Comment by liangxia** [comment]
This branch cannot be rebased due to conflicts
Rebasing the commits of this branch on top of the base branch cannot be performed automatically due to conflicts encountered while reapplying the individual commits from the head branch. 


Please rebase and squash your PR.

**Comment by zhaozhanqi** [comment]
> This branch cannot be rebased due to conflicts
> Rebasing the commits of this branch on top of the base branch cannot be performed automatically due to conflicts encountered while reapplying the individual commits from the head branch.
> 
> Please rebase and squash your PR.

@liangxia  Conflicts resolved.  please help take a look, thanks

File: features/step_definitions/sriov.rb [hunk 3]
```diff
@@ -116,6 +116,21 @@
...
+    end
+
+    step %Q/I run oc create over "#{resource_yaml}" replacing paths:/,table(%{
+      | ["spec"]["networkNamespace"] | "#{cr_project}" |
+    })
+
+    step %Q{the step should succeed}
+    teardown_add {
+      step %Q/I delete the "#{cr_name}" sriovibnetwork/
+    }
```
Comment (liangxia) [inline features/step_definitions/sriov.rb:133]: Can be simply with `step %Q/admin ensures "xxx_resource_name_xxx" yyy_resource_type_yyy is deleted from the "zzz_project_name_zzz" project after scenario/`

Comment (zhaozhanqi) [reply features/step_definitions/sriov.rb:133]: updated. @liangxia  
Could you help take a look, again.

**Comment by zhaozhanqi** [comment]
@liangxia Would you like to take a look if this can be merged, thanks.

**Comment by zhaozhanqi** [comment]
@pruan-rht @jhou1 @liangxia Could you help check if this can be merged?

**Pull request merged**